### ISSUE TITLE
a memory leak on printer desktop

### ISSUE THREAD

**Issue by NullYing** [mhammond/pywin32#1483]
Note that issues in this repository are only for bugs or feature requests in the pywin32.

**If you need support or help using this package, please follow [these instructions](https://github.com/mhammond/pywin32/blob/master/README.md#support)** - support or help requests will be closed without comment.

For all bugs, please provide the following information.

* Expected behavior and actual behavior.

 DeleteDC no free the memory

* Steps to reproduce the problem.

```python
@profile(precision=10)
def get_device_context(printer_name):
    p = win32print.OpenPrinter(printer_name)
    printer = win32print.GetPrinter(p, 2)
    devmode = printer["pDevMode"]
    print_processor = printer["pPrintProcessor"]
    pDC = win32gui.CreateDC(print_processor, printer_name, devmode)
    win32gui.DeleteDC(pDC)
    win32print.ClosePrinter(p)
```

loop 1 time

![image](https://user-images.githubusercontent.com/10618381/76479344-6dbfdc00-6446-11ea-934a-83925a1290ac.png)

loop 500 time after

![image](https://user-images.githubusercontent.com/10618381/76479420-abbd0000-6446-11ea-88a3-7a8e953c9792.png)

* Version of Python and pywin32

227

**Comment by mhammond** [comment]
I don't think this is a problem with pywin32. I can get a small amount of memory leaked when using my local printer, and doesn't fail before I kill it at about 20,000 iterations. I can also get a small amount of memory leaked in a loop which does nohting other than:

    pDC = win32gui.CreateDC("DISPLAY", None, devmode)
    win32gui.DeleteDC(pDC)

And inspecting the source to pywin32 I can't see a leak there - however, googling shows others struggle with CreateDC leaking memory too.
